tests/unit/bokeh/core/property/test_visual.py

```diff
@@ -273,6 +274,20 @@ def test_transform_data_url(self) -> None:
         image = "data:image/png;base64,"
         assert prop.transform(image) == image
 
+    def test_transform_path(self) -> None:
+        temp_dir = Path(tempfile.gettempdir())
+        with tempfile.NamedTemporaryFile() as file:
+            location = temp_dir / file.name
+        image = PIL.Image.new('RGBA', size=(50, 50), color=(155, 0, 0))
+        image.save(location, 'png')
+        prop = bcpv.Image()
+        assert prop.transform(location).startswith("data:image/png")
```

Comment:
This looks wrong to me. `file.name` is already a full path to the system temp directory (e.g. `/tmp`). We also want it to be preserved for as long as we need it. Something like this would be better:
```suggestion
    def test_transform_path(self) -> None:
        with tempfile.NamedTemporaryFile() as file:
            path = Path(file.name)
            image = PIL.Image.new('RGBA', size=(50, 50), color=(155, 0, 0))
            image.save(file, 'png')
            prop = bcpv.Image()
            assert prop.transform(path).startswith("data:image/png")
```

Reply:
@osule I agree with @mattpap can you apply and quickly test out the suggestion above then we should be able to get this merged

Reply:
For the sake of getting this merged, I am going to go ahead and apply the suggestion above, and then merge, assuming the results are green. If not, we can circle back :)